docs/03_start_the_encoding_workflow.md

```diff
@@ -48,14 +48,16 @@ In Cloud Shell, run:
 
 ```bash
 gcloud pubsub topics publish encoder-topic-[EVENT_ID] \
-    --message='{"truckOriginIp": "[TRUCK_IP]", "eventId": "[EVENT_ID]", "region": "[REGION]"}'
+    --message='{"truckOriginIp": "[TRUCK_IP]", "eventId": "[EVENT_ID]", "region_primary": "[REGION_PRIMARY]", "port_primary":"[PORT_PRIMARY]", "region_backup": "[REGION_BACKUP]", "port_backup":"[PORT_BACKUP]"}'
 ```
 
 Replace the following:
 
 -  `[EVENT_ID]` is a unique name for a streaming event.
 -  `[TRUCK_IP]` is the external IP address of the Gateway instance.
--  `[REGION]` is the same region as your GKE deployment.
+-  `[REGION_*]` is the same region as your GKE deployment.
+-  `[PORT_*]` is the same port as you configured on the Haivision gateway.
+
```

Comment:
I know it seems redundant, but I would make this 4 bullets where you specifically spell out the four parameters, REGION_PRIMARY, REGION_BACKUP, PORT_PRIMARY, PORT_BACKUP